lib/uktest/include/uk/test.h

```diff
@@ -694,7 +694,8 @@ _uk_test_do_assert(struct uk_testcase *esac, unsigned short line, int cond,
 #define UK_TEST_EXPECT(cond)						\
 	UK_TEST_ASSERTF(						\
 		(cond),							\
-		"expected `" STRINGIFY(cond) "` to be true"		\
+		"expected `%s` to be true", \
+		STRINGIFY(cond)		\
```

Comment:
Please make sure that the trailing `\` is aligned with the other backslashes. Same for the other locations you changed.

```suggestion
		"expected `%s` to be true",				\
		STRINGIFY(cond)						\
```

Reply:
Done. Hope I understood it correctly. Is it ok now?

Reply:
Hi @maniatro111, almost. But you probably have your editor configured wrong. Set it to 1 tab = 8 spaces. Optimally, set it to remove trailing blank characters and to show blank characters (tabs + spaces) so you do not mix up. Also configure a ruler at column 80. As you can see checkpatch is not happy because your lines are now exceeding the 80 characters limit.

Ensure that you keep the `\` at the column where it was before your changes (i.e., 73). 

Is:
![image](https://user-images.githubusercontent.com/8928900/184585809-926fc42a-6311-4b76-a26a-697e7355d75a.png)

Should be:
![image](https://user-images.githubusercontent.com/8928900/184585988-48c8ef5d-c868-4bb7-ae9b-00b87d1be5d9.png)

Reply:
Should be ok now.